build-tools/scripts/TestApks.targets

```diff
@@ -26,6 +26,7 @@
     <AvdLaunchTimeoutMinutes Condition=" '$(AvdLaunchTimeoutMinutes)' == '' ">5</AvdLaunchTimeoutMinutes>
     <AvdLaunchTimeoutSeconds>$([MSBuild]::Multiply($(AvdLaunchTimeoutMinutes), 60))</AvdLaunchTimeoutSeconds>
     <AvdLaunchTimeoutMS>$([MSBuild]::Multiply($(AvdLaunchTimeoutSeconds), 1000))</AvdLaunchTimeoutMS>
+    <_EmulatorImageType Condition=" '$(_EmulatorImageType)' == '' ">google_apis</_EmulatorImageType>
```

Comment:
I think we could instead make these changes at the top of this prop group:
```diff
-    <SdkManagerImageName Condition=" '$(SdkManagerImageName)' == '' ">system-images;android-$(TestAvdApiLevel);default;$(TestAvdAbi)</SdkManagerImageName>
+    <TestAvdType Condition=" '$(TestAvdType)' == '' ">google_apis</TestAvdType>
+    <SdkManagerImageName Condition=" '$(SdkManagerImageName)' == '' ">system-images;android-$(TestAvdApiLevel);$(TestAvdType);$(TestAvdAbi)</SdkManagerImageName>
```
With those changes you shouldn't need to pass `ImageType`  to `<CreateAndroidEmulator/> below, since `TargetId` is being used. I had this almost working locally, though for some strange reason the new test result XML had some invalid characters at the start that _may_ need to be trimmed?

```xml
ï»¿<?xml version="1.0" encoding="utf-8" standalone="no"?>
<!--This file represents the results of running a test suite-->
<test-results name="Mono.Android_Tests" total="861" errors="0" failures="0" not-run="2" inconclusive="0" ignored="2" skipped="0" invalid="0" date="2021-02-02" time="20:43:56">
  ```

We'll also now need to consider the new nightly job and make sure the test runner changes will still work on the older emulators.